### ISSUE TITLE
Can't load any .md file after closing all tabs on first run after fresh install

### ISSUE THREAD

**Issue by ailintom** [Zettlr/Zettlr#4266]
### Description

I encounter a curious behaviour with Zettlr 3.0.0 Beta 3 on Windows 11. When I close all the default tabs when I first run the program upon a fresh install, I cannot load any file from any workspace, unless I restart Zettlr. 

### Reproducing

1. Make a fresh install of Zettlr 3.0.0 Beta 3 on Windows 11.
2. Start the program, it will show you a number of default tabs from the Zettlr Tutorial workspace. 
3. Close all open tabs
4. Try to reopen any of the files from the workspace. It won't open (it does not change if you open a new workspace). No new entries appear in logs when you try open any .md file.
5. Restart Zettlr, it works normally

### Zettlr Version

Beta (if applicable)

### Specify version

3.0.0 beta 3

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 11

### Additional Information

Not really a big deal for experienced users, but a major disappointment for first-time users who have just installed the program, and it stops working on first run.

**Comment by nathanlesage** [comment]
> Not really a big deal for experienced users, but a major disappointment for first-time users who have just installed the program, and it stops working on first run.

That's why it's still in beta ;)

But sounds like a curious bug, I'll have to check that.

**Comment by nathanlesage** [comment]
I just tried it out but I couldn't reproduce the bug. Maybe it was an issue in the beta that got fixed alongside something else since then ...? Feel free to try out the upcoming nightly and see if the problem persists.

**Comment by ailintom** [comment]
Will try. I just wanted to add that I use Windows Sandbox to try Zettlr beta, so that there are no rests of any previous installations of Zettlr in the system. And this problem only occurs on first run. (Which is every time I run Sandbox anew and install beta 3)

**Comment by nathanlesage** [comment]
I heard of Windows Sandbox for the first time today. I cannot rule out that there might be an interaction. I would suspect that possibly the issue is not appearing only after you close the tabs, but rather already exists before that (i.e. even without closing tabs you may be unable to open new files).

**Comment by ailintom** [comment]
Well, I've tried yesterday's nightly and the wrong behaviour is still there.
![Animation](https://user-images.githubusercontent.com/24455000/232687959-a883207d-e8dd-40a6-b4b9-4c098815089e.gif)

**Comment by nathanlesage** [comment]
Ouh, I see what is happening. This issue is a duplicate of #4033

Closing as duplicate.

**Comment by nathanlesage** [comment]
In your screengrab you can exactly see that there is a ghost tabbar in the middle of the empty editor sense, so I am pretty certain that yours is simply an edge-case of that.

**Comment by ailintom** [comment]
@nathanlesage Problem no longer occurs in Beta 4. Thank you!
